packages/tools/src/eventListeners/touch/touchStartListener.ts

```diff
@@ -104,9 +104,13 @@ const defaultState: ITouchStartListenerState = {
 
   swipeDistanceThreshold: 48,
   swiped: false,
-  swipeWindow: 200, // 200ms to swipe after touch start or no swipe trigger
+  swipeToleranceMs: 300, // user has 300ms to swipe after touch start or no swipe will trigger
```

Comment:
The new value does not help my phone, nor another Android phone I tried and a Chromebook that I tried. So feel free to revert this change, but it is ok as it as well. Up to you. Thanks for trying it for me.

Reply:
hmmm that's really strange, if I manually open a browser window and go into mobile mode it can trigger

Reply:
Well maybe the new tolerance is slightly better, but it still feels difficult to trigger. Let's just leave this as is for now.